go/kbfs/libkbfs/config_local.go

```diff
@@ -216,7 +216,7 @@ var _ Config = (*ConfigLocal)(nil)
 
 // getDefaultCleanBlockCacheCapacity returns the default clean block
 // cache capacity. If we can get total RAM of the system, we cap at
-// the smaller of <1/4 of available memory> and
+// the smaller of <1/8 of available memory> and
```

Comment:
Could you explain where 1/8 comes from? It seems if we assume Bleve uses 100x+, by using a factor of 500 we are capping at more than 1/5.

Reply:
This is just a comment correction to the existing config_local block cache memory computation.  The code uses `/8`; the comment was wrong.  It's unrelated to the rest of the PR, sorry, I just noticed it when I was looking through that code.  I can split it out into its own PR if you want.

Reply:
Ah, got it. No need to split -- we'll have this comment to remind us if we ever get confused. Thanks! LGTM